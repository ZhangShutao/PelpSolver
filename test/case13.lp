a1 | a2 | a3. [1]
l :- K(0, 1) a1.
:- l.