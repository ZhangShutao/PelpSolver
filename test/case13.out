WorldView 1:
{[1.0]:{a1}}
WorldView 2:
{[0.4223187982515182]:{a3},[0.15536240349696362]:{},[0.4223187982515182]:{a2}}